utilities/metadata_parser.py

```diff
@@ -289,7 +298,7 @@ def page_image_to_doc(folder_name, pdf_path, image_directory):
             pass
 
 
-def interpret_person_organization(field, item_organization, item_person, new_doc):
+def interpret_person_organization(field, item_organization, item_person, new_doc, line_no=None, names_so_far={}):
```

Comment:
Looks like we have some errors coming out of this bc we're splitting on `'; '` and `', '` (with spaces after) rather than ';' and ','.

Could you adjust the split to happen on just the punctuation, and then strip whitespace after? See, e.g.,

`First name here for last name Corbato is F. J.. Did you mean F. J. ;McCarthy?`

McCarthy is clearly the next last name, but it's getting split incorrectly.